overlay/base/usr/lib/system/_os/podman.py

```
@@ -331,8 +358,10 @@ def image_name_from_parts(
 
 def image_qualified_name(image: str) -> str:
     registry, repo, tag, digest = image_name_parts(image)
-    if ((registry or REGISTRY) == REGISTRY) and repo == IMAGE:
+    if ((registry or REGISTRY) == REGISTRY) and repo == IMAGE or repo == OS_NAME:
```

Ruff (0.16.2)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)